build.sbt

```diff
@@ -156,6 +157,11 @@ lazy val inject = (project in file("modules/inject"))
   .settings(name := "trace4cats-inject", libraryDependencies ++= Seq(Dependencies.catsEffect).map(_ % Test))
   .dependsOn(core, base)
 
+lazy val `cats-effect` = (project in file("modules/cats-effect"))
+  .settings(publishSettings)
+  .settings(name := "trace4cats-cats-effect", libraryDependencies ++= Seq(Dependencies.catsEffect))
```

Comment:
IMO this naming is misleading. t4c is already fully based on `cats-effect` typeclasses.

I would name it `trace4cats-cats-effect-io`, or just `trace4cats-cats-effects-iolocal`

Reply:
Renamed to `trace4cats-cats-effect-io`